### DISCUSSION TITLE
Fluctuating growth ratios of coculture with different glucose uptakes

### DISCUSSION THREAD

**Discussion by maziya** [micom-dev/micom#125] [Q&A] [answered]
I have simulated a coculture with the iAF1260 and iMM904 models and run MICOM with the cooperative tradeoff of 1 and iteratively varying the uptake of glucose in the medium from 1 to 30 at intervals of 0.5. The growth ratios of the two models look like this. 
 
![ratio](https://github.com/micom-dev/micom/assets/42278922/6b3c214c-a002-485f-8337-aff064fad6df)

Is this expected from MICOM? There is too much fluctuation in the growth values. Ideally, it should either increase or decrease monotonically.

**Comment by cdiener** [comment, marked as answer]
Hmm, interesting. So it's not impossible because cooperative tradeoff FBA is a non-linear method, so you don't necessarily see monotonic behavior here. But even without that I could imagine cases where that behavior would occur. For instance, if one taxon A needs some cofactor to efficiently grow on glucose and taxon B does not. If that cofactor is limited in the medium you would first see taxon A dominate in lower glucose ratios and then have B catch up as there is more glucose and the cofactor is insufficient.
